### PR TITLE
build(checkver): Fix output with '-Version'

### PR DESCRIPTION
**Pull request by niheaven**
Before

wrong

After

right

### CODE REVIEW

**Comment by Ash258** [comment]
If I can see correctly, main changes are:

- https://github.com/lukesampson/scoop/pull/3774/files#diff-6d5ec5a5587d8c2e2cd120423846e581L308-L310
- https://github.com/lukesampson/scoop/pull/3774/files#diff-6d5ec5a5587d8c2e2cd120423846e581R78-R82
- https://github.com/lukesampson/scoop/pull/3774/files#diff-6d5ec5a5587d8c2e2cd120423846e581R202-R206

Rest is just messy compare due to nested `if (!$ver) {`, which was not changed right?

**Comment by r15ch13** [comment]
The same logic could be archived without the indentation by adding the check for `$Version` later.


Change
---
https://github.com/lukesampson/scoop/blob/a9fa775d59b14e7dce335313faa0eff855469764/bin/checkver.ps1#L274-L280

To
---
```pwsh
    if (!$ver) {
        next "couldn't find new version in $url"
        continue
    }

    if($Version) {
        $expected_ver = $ver
        $ver = $Version
    }

    # Skip actual only if versions are same and there is no -f
    if (($ver -eq $expected_ver) -and !$ForceUpdate -and $SkipUpdated) { continue }

```

**Comment by niheaven** [comment]
@Ash258 Yes, rest is just caused by indenting

@r15ch13 The big `if` just skips time-consuming version finding step.

**Comment by niheaven** [comment]
Tested after resolve conflicts (merged current 'develop' branch)

File: bin/checkver.ps1 [hunk 1]
```diff
@@ -15,6 +15,8 @@
     Useful for hash updates.
 .PARAMETER SkipUpdated
     Updated manifests will not be shown.
+.PARAMETER Version
+    Updated manifest to specific version.
```
Comment (rashil2000) [inline bin/checkver.ps1:19]:
```suggestion
    Update manifest to specific version.
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
Will it be possible to isolate indentation changes in a separate commit (in this PR itself)? Otherwise it's difficult to review.

**Comment by niheaven** [comment]
Separate into two commits, Part I is just indenting, while Part II is fixed code.

**Comment by Ash258** [comment]
image

**Comment by niheaven** [comment]
> image

Wonderful feature that I never found in GitHub.

**Comment by rashil2000** [comment]
> image

Right, completely forgot about this 😅

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**